frontend/components/explore/dark-mode.tsx

```
@@ -5,7 +5,7 @@ import { useTheme } from "next-themes";
 import { Sun, Moon } from "lucide-react";
 
 export function DarkMode({ className }: { className?: string }) {
-  const { theme, setTheme } = useTheme();
+  const { theme, setTheme, resolvedTheme } = useTheme();
```

ESLint

[error] 'theme' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)